fileFormat/ResourceFiles.md

```diff
@@ -6,14 +6,14 @@ Resource files are structured archives. These files contain one or more "resourc
 A resource identifier is a ```uint16``` value, and a compound resource uses a ```uint16``` value as index.
 
 Resource identifier are "valid" from value ```0x0003``` and above.
-```0x0000``` is the NULL identifier, and 1 and 2 are used engine-internally.
+```0x0000``` is the NULL identifier (it means that this resource will be erased from file), and ```0x0001``` and ```0x0002``` are used engine-internally.
```

Comment:
What do you mean with "resource will be erased from file"? Because savgames have each a resource with ID 0x0000 in them. (Probably not intended, yet, they exist).
Please clarify.

Reply:
0x0000 meant to be quick approach for developers to remove unused assets from res. They left in source code bunch of tools, on of them - resbuild has [ResPack()](https://github.com/winterheart/systemshock/blob/master/src/Libraries/RES/Source/resbuild.c#L359) function that removes holes in RES file. It checks if id == 0 and erase it from file.

Reply:
Ok, if this is only in the tool, then please point it out. The specs are here for the main engine.